lib/controllers/base.js

```diff
@@ -133,11 +147,7 @@ class Controller {
     response.writeHead(status, headers);
     response.end(html);
     const username = locals.username || (locals.params && locals.params.username) || '-';
-    logRequest(this.request, username, status, html.length, locals.message || locals.error);
-  }
-
-  errorPage (status, message) {
```

Comment:
Wouldn't it be better to keep the shorthand function and just extend it a bit, rather than removing it entirely and making the error page call more error-prone?

Reply:
I've added a more specific title and message to each call, so the only value `errorPage` is supplying is 'error.html'. But we can have `errorPage` supply default values for title and message.